packages/E2ETest/windows/ReactUWPTestApp/MainPage.xaml.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.ReactNative;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace ReactUWPTestApp
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+            this.InitializeComponent();
+            var app = Application.Current as App;
+            myRootView.ReactNativeHost = app.Host;
+            app.Host.ReloadInstance();
```

Comment:
>app.Host.ReloadInstance(); [](start = 12, length = 26)

It should not be here. We want to load instance before the window is created. Reloading instance on UI creation will force us to make UI loading slower.